test/extended/util/test.go

```diff
@@ -256,6 +256,7 @@ var (
 
 		// Doesn't work on scaled up clusters
 		`\[Feature:ImagePrune\]`,
+		`\[Feature:ImageMirror\]`,
```

Comment:
why wouldn't this work on a scaled up cluster?  you appear to be standing up the new registries in pods.

Reply:
Because of the new approach with the launch of everything in pods, this test have become not just slow, but extremely slow.

Reply:
how slow?
and that's what we have the [Slow] tag for

Reply:
Few times slower on my laptop. I have this tag. This tag in list [too](https://github.com/openshift/origin/pull/19834/files/fb45012a304d133967908a2c62736b7de7012f34#diff-1985610d6999ca5f06e0e2ca4d9c27e0R249).